### DISCUSSION TITLE
Icon slight redesign.

### DISCUSSION THREAD

**Discussion by DuttaAB-dev** [Heroic-Games-Launcher/HeroicGamesLauncher#531] [Ideas]
The icon looks too boxy. Maybe make the edges a bit rounded to better suite desktop environments like GNOME?

**Comment by Flonky** [comment]
First thing id fix is the sword being slightly off center

I've noticed that when i was working with it on figma

**Comment by ghost** [comment]
Why not simplify the logo? I think that it has too many gradients.
Something like this: https://www.figma.com/file/X8hP94D0CPrOO83TNzJBaA/Untitled

**Comment by flavioislima** [comment]
Yeah, we will never find a logo that will make everyone happy.
I think we need to improve the ones that stay on the tray icon, that should be the priority in my opinion.

**Comment by flavioislima** [comment]
Converting to a discussion.
